src/components/Divider/docs/Divider.mdx

```diff
@@ -0,0 +1,88 @@
+---
+name: Divider
+menu: Components
+route: /components/divider
+---
+
+import { Playground } from 'docz';
+import { ItemWrapper } from '../../../../docs/components/ItemWrapper.ts';
+import { Divider } from '../Divider.tsx';
+import { WrappedHorizontalDivider, WrappedVerticalDivider } from './WrappedDivider.tsx';
+
+# Divider
+
+**Primary UI element for visually separating content**
+
+Renders a divider UI component: horizontal or vertical line that visually separates two pieces of data, content or UI.
+<br/>
+<hr/>
+
+### Default Behaviour
+Horizontal divider takes up full available width, vertical divider takes up full available height.
+
+Horizontal divider is rendered by default. Set **vertical** prop to __true__ to change divider orientation.
+<br/>
+<hr/>
+
+### Divider vs Border
+
+The default color of Divider is **$border.primary** (#C6CDD4), however...
+
+**Divider is NOT a border, and should not be used as such. Please do not use this component as a border for elements.**
+
+Divider is naturally expected to have a certain offset from the elements it is 'dividing' or separating.
+<br/>
+<hr/>
+
+### Style Props
+
+The Divider has following design props:
+- **offset** - set the divider offset from the content it is separating (uses _mx_, _my_ styled system props)
+
+<hr/>
+
+## Usage 
+
+### Horizontal (without offset)
+
+<ItemWrapper>
+    <WrappedHorizontalDivider offset={0} />
+</ItemWrapper>
+
+```jsx
+<Divider />
+```
+
+<hr/> 
+
+### Horizontal (with offset)
+
+<ItemWrapper>
+    <WrappedHorizontalDivider offset={5} />
```

Comment:
I think I'd like to see an example with a margin-top different from the bottom as it's the way I saw it used the most.

Reply:
We're still having active discussion about how margins will work on the issue, however, I'm not sure if we will even manage top and bottom margin separately for Divider: https://github.com/freenowtech/wave/issues/218.

Could you link an example or screenshot of that case you mentioned, meanwhile please? Is it really a Divider if it has different offset from the two items it is positioned between? Is it not more of a border, rather?

Reply:
@div-Leo , @JanHamara in design we use a lot of different top/bottom margins. But since we have in addition spacers we can simplify the dividers and consider having the same top/bottom spacing and in the case when we need to have a different from one of the sides - use an additional spacer.

Reply:
I saw spacers used in the layout but I don't think we have the equivalent programmatically (in Wave).

Reply:
> Could you link an example or screenshot of that case you mentioned, meanwhile please? Is it really a Divider if it has different offset from the two items it is positioned between? Is it not more of a border, rather?

Check "Advanced search" form, we have Dividers with different margin top/bottom
<img width="945" alt="Screenshot 2022-05-04 at 14 30 23" src="https://user-images.githubusercontent.com/12762609/166681499-966ab5e4-70cc-45a7-b997-9883dea8dc5f.png">

Reply:
I agree with Rafael's solution, I think the example in Advanced Search you linked it isn't really a goal to have a divider with different margins, if you look at the other collapsible items it's visible that there is some additional whitespace after the form (like above Hailing Type section above the highlighted yellow area) and because of that whitespace there is a smaller top margin, because we actually want to have same spacing both top and bottom of the line. So in this case, the additional whitespace above yellow area + top-margin **is (almost) same as** bottom-margin. 

So normally, we wouldn't have any padding after the select input field, and we would have a divider that equally separates both sections (as according to couple design and composition principles like alignment and proximity).

@rafael-sepeda correct me if I'm wrong, but that's the composition I would expect, unless there is particular reason why one section would have different offset from divider than the other?

<img width="1295" alt="Screenshot 2022-05-04 at 15 52 30" src="https://user-images.githubusercontent.com/14894844/166696899-ce02ca9f-c3dc-4687-b2b9-5e09a3ec4cba.png">

Reply:
"(almost)" still means different...  Honestly, I don't get what you're trying to say. What you say is the top margin is the bottom margin of the input, I guess what matters is what we have in the code not what we see... 

Anyway, this is something that only @rafael-sepeda can answer, so I'm going to leave it to you guys.

Reply:
> I saw spacers used in the layout but I don't think we have the equivalent programmatically (in Wave).

@div-Leo we do have them in Wave:
https://github.com/freenowtech/wave/tree/main/src/essentials/Spaces
https://wave.free-now.com/essentials/spaces

In Figma we also have components for them that use same specs.

Reply:
> correct me if I'm wrong, but that's the composition I would expect, unless there is particular reason why one section would have different offset from divider than the other?
Sometimes we need to have different spacings from dividers if the dividers show the end of one section and you need to have a bigger spacing after to show a more clear start for the next section. But more often the top/bottom spacing is the same, especially if this is part of a component or a list of components, so, from my perspective for simplicity, we can set one margin value for top/bottom for spacers in dividers and then use additional spacers if needed.

Reply:
Ok, thank you @rafael-sepeda, I will **resolve** this conversation.